wolfcrypt/benchmark/benchmark.c

```diff
@@ -6404,6 +6422,97 @@ void bench_sakke(void)
 #endif /* WOLFCRYPT_SAKKE_CLIENT */
 #endif /* WOLFCRYPT_HAVE_SAKKE */
 
+#ifdef HAVE_LIBOQS
+void bench_falconKeySign(byte level)
+{
+    int    ret = 0;
+    falcon_key key;
+    double start;
+    int    i, count;
+    byte   sig1[FALCON_LEVEL1_SIG_SIZE];
+    byte   sig5[FALCON_LEVEL5_SIG_SIZE];
+    byte   msg[512];
```

Comment:
Do we have a macro now for `sig` max size?

Reply:
We do, but by doing this I get to use sizeof() below which looks neater to me.

Reply:
yeah...alright...I'll change.